cmd/utils.go

```diff
@@ -1491,13 +1495,17 @@ func GenServiceYaml(log *LoggingConfig, appName string, ports []string, pdir str
 	service.Spec.ServiceType = "NodePort"
 	service.Spec.Ports = make([]Port, len(ports))
```

Comment:
this line is creating an empty port field in the case where we have a debug port. The len(ports) will include the debugport, but during the for loop we will not set those values.